community/cypher/cypher-compiler-3.0/src/test/scala/org/neo4j/cypher/internal/compiler/v3_0/planner/logical/greedy/CartesianProductTest.scala

```diff
@@ -1,70 +0,0 @@
-/*
- * Copyright (c) 2002-2015 "Neo Technology,"
- * Network Engine for Objects in Lund AB [http://neotechnology.com]
- *
- * This file is part of Neo4j.
- *
- * Neo4j is free software: you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation, either version 3 of the License, or
- * (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program.  If not, see <http://www.gnu.org/licenses/>.
- */
-package org.neo4j.cypher.internal.compiler.v3_0.planner.logical.greedy
-
-import org.mockito.Mockito._
-import org.neo4j.cypher.internal.compiler.v3_0.HardcodedGraphStatistics
-import org.neo4j.cypher.internal.frontend.v3_0.ast.PatternExpression
-import org.neo4j.cypher.internal.compiler.v3_0.planner.logical.Cost
-import org.neo4j.cypher.internal.compiler.v3_0.planner.logical.Metrics.QueryGraphSolverInput
-import org.neo4j.cypher.internal.compiler.v3_0.planner.logical.plans.{CartesianProduct, LogicalPlan}
-import org.neo4j.cypher.internal.compiler.v3_0.planner.{LogicalPlanningTestSupport, QueryGraph}
-import org.neo4j.cypher.internal.frontend.v3_0.test_helpers.CypherFunSuite
-
-class CartesianProductTest extends CypherFunSuite with LogicalPlanningTestSupport {
```

Comment:
I think the tests in CartesianProductsOrValueJoinsTest and CartesianProductPlanningIntegrationTest seem to cover this for IDP.